sdk/core/azure-core/azure/core/pipeline_client.py

```diff
@@ -60,7 +60,7 @@ class PipelineClient(PipelineClientBase):
     Builds a Pipeline client.
 
     :param str base_url: URL for the request.
-    :keyword Configuration config: If omitted, it will be used by the PipelineClient.
+    :keyword Configuration config: If omitted, the config object will be used by the PipelineClient.
```

Comment:
Sorry, I'm still confused by this. If the `config` object is omitted, how is it *also* what will be used? I think this is what happens based on looking at the code:

> :keyword Configuration config: If omitted, a Configuration object created from additional keyword arguments will used by the PipelineClient.

but please let me know if that's off base.

Reply:
You are absolutely right!